src/PackageManagement.VisualStudio/ScriptPackageFile.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Runtime.Versioning;
+using NuGet.Frameworks;
+
+namespace NuGet.PackageManagement.VisualStudio
+{
+    public class ScriptPackageFile : IScriptPackageFile
+    {
+        private readonly NuGetFramework _targetFramework;
+
+        public ScriptPackageFile(string path, NuGetFramework targetFramework)
+        {
+            Path = path.Replace(System.IO.Path.AltDirectorySeparatorChar, System.IO.Path.DirectorySeparatorChar);
+            _targetFramework = targetFramework;
+        }
+
+        public string Path
+        {
+            get;
+            set;
+        }
+
+        public FrameworkName TargetFramework
+        {
+            get
+            {
+                return new FrameworkName(_targetFramework.DotNetFrameworkName);
```

Comment:
Since, _targetFramework is read-only, why don't you simply have a readonly FrameworkName property and compute it once in the constructor, instead of having a readonly NuGetFramework field and a non-automatic property of type FrameworkName. Fewer lines